config/config.go

```diff
@@ -22,6 +26,7 @@ type Config struct {
 	users         []string
 	apiToken      string
 	targetURLs    []string
+	gitHubApp     bool
```

Comment:
this looks to be defined but never used. would be good to set this during init then reuse this instead of having to call `os.Getenv` for each use